### PR TITLE
Polkadot Documentation

### PR DESCRIPTION
**Pull request by mkchungs**

### CODE REVIEW

File: data-catalog/substrate/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    A2.ticker as ticker_2,
+    method
+    FROM hydradx.events E
+    LEFT JOIN query_3482301 A1 on A1.asset_id=CAST(JSON_VALUE(data, 'strict $[1]') as INT)
+    LEFT JOIN query_3482301 A2 on A2.asset_id=CAST(JSON_VALUE(data, 'strict $[2]') as INT)
+    WHERE section='omnipool' and method in('SellExecuted', 'BuyExecuted')
+```
+
+As Substrate chains uses SS58 Addresses, the following Dune function are very useful:
+* [SS58 functions: from_ss58(varchar) + to_ss58(varbinary, int)](https://docs.dune.com/query-engine/Functions-and-operators/ss58)
```
Comment (hawkaa) [inline data-catalog/substrate/overview.mdx:147]: Are there ways to use relative links instead of absolute ones?

**Review by hawkaa — COMMENTED** [review]
Looks great on my end, but need @0xBoxer to give the final thumbs up.

**Comment by 0xBoxer** [comment]
@mkchungs Please enable "allow edits by maintainers" for this PR.

**Comment by mkchungs** [comment]
@0xBoxer: I don't see such option, can you please advise?

**Comment by 0xBoxer** [comment]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

usually should be able to do this 🤔

**Comment by mkchungs** [comment]
@0xBoxer  This PR is not user-owned but rather created directly within our organization. I don't think see the "allow edits by maintainers" option available.

I've invited you as admin to our fork at [colorfulnotion:polkadot_substrate](https://github.com/colorfulnotion/dune-api-docs/tree/polkadot_substrate) - this should enable you to all the edits you need. can you please check?

Screenshot 2024-03-15 at 8 00 47 PM

**Comment by 0xBoxer** [comment]
I made quite a few changes and fixes to improve and fit our usual docs structure and language, this is good to merge now from my POV.

**Pull request merged**